src/components/dialogs/StandardDialog.svelte

```diff
@@ -36,7 +37,10 @@
 <div class="relative z-10">
   <Transition show={$dialog.expanded}>
     <div
-      class="fixed top-0 left-0 h-screen w-screen flex justify-center items-center bg-black/50 bg-blend-darken">
+      class="fixed top-0 left-0 h-screen w-screen flex justify-center items-center bg-black/50 bg-blend-darken"
+      use:dialog.modal
+      role="none"
```

Comment:
I know it complains, but this element dynamically gets the correct role of "dialog", I don't think we should be overwriting that to "none" here.